src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -36,6 +36,8 @@ namespace System.Globalization
 #endif
     sealed partial class IdnMapping
     {
+        private const int StackallocThreshold = 512;
+
```

Comment:
I think I'd prefer to keep this local to the method that's using it, so that it's clear at the call site.  It's in a const simply to give a name to the number so that it doesn't appear to be magic.